src/appshell/qml/kdab/WindowContent.qml

```diff
@@ -23,7 +23,7 @@ import QtQuick 2.15
 import QtQuick.Controls 2.15
 
 import MuseScore.AppShell 1.0
-import MuseScore.Dock 1.0
+import MuseScore.Dock 1.0 as Dock
```

Comment:
Why did you decide to add an alias and use it?

Reply:
removed